TransformFound: false
InputMsg: "/kinect/depth/points"
DownSamplingRadius: 0.008
FeatureClusterMaxLength: 0.1
FeatureClusterMaxPoints: 200
NormalRadius: 0.05
FeatureRadius : 0.2
SamplingRadius : 0.02
FeatureScoreThreshold : 0.40
FilterRadius : 0.2
FilterVariance : 0.01
KeyPointRadius : 0.05
MaxMotion : 0.2
StaticFrontThreshold : 3.0
Verbose : true
StoreResults : false
#OutputFolder : "/home/dewan/squirrel/catkin_ws/src/squirrel_nav/squirrel_dynamic_filter/results/test/"
#InputFolder : "/home/dewan/squirrel/catkin_ws/src/squirrel_nav/squirrel_dynamic_filter/"
#InputFolder : "/home/alufr/catkin_ws/src/squirrel_nav/squirrel_dynamic_filter/"

